scripts/config/nconf.c

```diff
@@ -0,0 +1,1557 @@
+/*
```

Comment:
Can we pull `nconf.[hc]` from a URL instead of embedding them directly?

Reply:
LEDE/OpenWrt build phases are configuration, source downloads, tools compilation, toolchain compilation, target compilation, and firmware generation.

Adding download during configuration phase doesn't make sense.